alpine/generate-dockerfile.sh

```diff
@@ -56,10 +56,19 @@ for repo in $repos; do
 		dockerfilePath=$repo/$suite
 		mkdir -p $dockerfilePath
 
+		if version_le $suite "3.4"; then
```

Comment:
Because tini is not available in Alpine Linux 3.4, however I removed this since we dropped support for Alpine Linux 3.4